CONTRIBUTING.md

```diff
@@ -12,6 +12,12 @@ After creating a pull request, assign one of the Nation3 [teams](https://github.
 
 When you have 2 approved reviews, merge the pull request.
 
+## Test Network
+We use Goreli for our testnet deployemnts deployments. You can get some testnet ETH from the following faucets:
```

Comment:
```suggestion
We use Goerli for our testnet deployments. You can get some testnet ETH from the following faucets:
```

Reply:
You can tell I am not a proof reader, even now I have to concentrate to see what the differences are! Maybe I should stick to code.